website/pages/docs/plugins/destinations/azblob/overview.mdx

```diff
@@ -68,6 +67,18 @@ This is the (nested) spec used by the Azure blob destination Plugin.
 
   Optional parameters to change the format of the file
 
+- `batch_size` (integer, optional. default: 10000)
+
+  Number of records to write before starting a new object.
+
+- `batch_size_bytes` (integer, optional. default: 52428800 (50 MiB))
+
+  Number of bytes (as Arrow buffer size) to write before starting a new object.
+
+- `batch_timeout_ms` (integer, optional. default: 30000 (30 seconds))
```

Comment:
Can we change this to `batch_timeout` with `configtype.Duration` backing it like in Postgres and ClickHouse? 😬  (Here and in the other file plugins) `configtype.Duration` will be available soon in SDK `v4.7.0-rc1`

Reply:
yeah I'll update the plugins shortly

Reply:
Done